packages/cli/src/license/license.controller.ts

```diff
@@ -73,11 +73,28 @@ licenseController.post(
 		// Call the license manager activate function and tell it to throw an error
 		const license = getLicense();
 		try {
-			console.log(await license.activate(req.body.activationKey));
+			await license.activate(req.body.activationKey);
 		} catch (e) {
-			if (e instanceof Error) {
-				throw new ResponseHelper.BadRequestError(e.message);
+			const error = e as Error & { errorId?: string };
+
+			switch (error.errorId ?? 'UNSPECIFIED') {
+				case 'SCHEMA_VALIDATION':
+					error.message = 'Activation key is in the wrong format';
+					break;
+				case 'RESERVATION_EXHAUSTED':
+					error.message =
+						'Activation key has been used too many times. Please contact sales@n8n.io if you would like to extend it';
```

Comment:
Product requirement are slightly different: 

`Activation key has been used too many times. Please <a mailto:sales@n8n.io>contact us</a> if you would like to extend it`

I removed the HTML markup as I didn't have time to figure out how to render it properly in the toast.

Reply:
I guess this is okay for now